index.d.ts

```diff
@@ -0,0 +1,14 @@
+import fastify, {
+  type FastifyBaseLogger,
+  type FastifyReply,
+  type FastifyRequest,
+} from "fastify";
+
+declare namespace fastifyasyncforge {
+  export const app: typeof fastify;
+  export const request: FastifyRequest;
+  export const reply: FastifyReply;
+  export const logger: FastifyBaseLogger;
```

Comment:
those are functions that returns this, not the actual thing. you might be missing some generics too.

Reply:
Done 👍🏼